sketch.js

```diff
@@ -1,18 +1,29 @@
 var amplitude, 
     level,
     song, 
-    speed;
+    speed,
+    amplitudeLevel,
+    count = 0;
 
 function preload(){
   song = loadSound('/assets/FortunateSon.mp3');
 }
 
 function setup() {
-  createCanvas(0, 0)
+  createCanvas(0, 0);
   song.play();
-  peaks = song.getPeaks([width]);
   amplitude = new p5.Amplitude();
-  level = amplitude.getLevel();
+
 }
 
-module.exports = level;
\ No newline at end of file
+function draw() {
+  level = amplitude.getLevel();
+  var size = map(level, 0, 1, 1, 30);
```

Comment:
takes in level as value mapped between 0 & 1, 
then expands range to map between 1 & 30